libraries.json

```diff
@@ -542,17 +539,17 @@
   {
     "github": "Liftric/KVault",
     "category": "Storage",
-    "maven": "https://dl.bintray.com/liftric/maven/com/liftric/kvault/"
+    "maven": "https://repo1.maven.org/maven2/com/liftric/kvault/"
   },
   {
     "github": "Liftric/Auth",
     "category": "Integration",
-    "maven": "https://dl.bintray.com/liftric/maven/com/liftric/auth/"
+    "maven": "https://repo1.maven.org/maven2/com/liftric/auth/"
   },
   {
     "github": "ACINQ/bitcoink",
     "category": "Integration",
-    "maven": "https://dl.bintray.com/acinq/libs/fr/acinq/bitcoink/bitcoink/"
+    "maven": "https://repo1.maven.org/maven2/fr/acinq/bitcoin/"
```

Comment:
at this path no library at all.
but here https://repo1.maven.org/maven2/fr/acinq/bitcoin/bitcoin-kmp/ has